concourse/pipelines/deploy.yml

```diff
@@ -291,9 +309,27 @@ jobs:
               -auto-approve
 
             terraform output -json > "$root_dir/govuk-terraform-outputs/govuk-terraform-outputs.json"
+
+            for app in content-store; do
+              terraform output -json "${app}" > "$root_dir/updated-${app}-terraform-outputs/${app}.json"
+
+              if cmp \
+                "$root_dir/original-${app}-terraform-outputs/${app}.json" \
+                "$root_dir/updated-${app}-terraform-outputs/${app}.json"
```

Comment:
semantics: I wonder if `new` is not better than `updated`

Reply:
`new` / `old` instead of `original` / `updated` ? I could be sold on that - they're a bit shorter at least.

Reply:
Resolved in 9a53cb4